releng/org.eclipse.glsp.repository/category.xml

```diff
@@ -67,6 +55,7 @@
       </description>
    </category-def>
    <category-def name="org.eclipse.glsp.dependencies" label="External Dependencies"/>
-   <repository-reference location="mvn:org.eclipse.jetty:jetty-p2:10.0.13:zip:p2site" enabled="true" />
-   <repository-reference location="https://download.eclipse.org/elk/updates/releases/0.8.1/" enabled="true" />
+   <repository-reference location="https://download.eclipse.org/elk/updates/releases/0.9.1/" name="ELK 0.9.1" enabled="true" />
```

Comment:
Should we also update the ELK version in the MANIFEST files or keep it at 0.8.1? Seems like we only build with 0.9.1 anymore.

Reply:
I don't think its necessary. Raising the minimum version without an explicit need might unnecessarily break adopters products. If you want we could adapt the target for the minimum version to use elk 0.8.1.
Then we are on the safe side an know that it works with both versions

Reply:
Perfect, just wanted to make sure, thank you!